includes/Plugin.php

```diff
@@ -250,13 +261,19 @@ public function register() {
 		add_action( 'init', [ $this->embed_base, 'init' ], 9 );
 
 		// Gutenberg Blocks.
-		$this->embed_block = new Embed_Block();
-		add_action( 'init', [ $this->embed_block, 'init' ] );
+		$this->web_stories_block = new Web_Stories_Block();
```

Comment:
There a chance of breakable here if were using this property to filter something in your plugin?

Reply:
I don't have a problem with that.

We've never declared these as extensible by others and [no plugin](https://wpdirectory.net/search/01EZS4CZW51ST8RM0F2PS0X7KT) is using this currently.

Don't need to bother with BC here.

Reply:
Agreed